doc/install-dependencies.md

```diff
@@ -55,6 +55,15 @@ sudo yum-config-manager --add-repo https://openresty.org/package/centos/openrest
 # install OpenResty and some compilation tools
 sudo yum install -y openresty curl git gcc luarocks lua-devel openresty-openssl-devel
 
+# if the luarocks installed by yum is not the 3.x, installing it from source.
+sudo bash ./utils/linux-install-luarocks.sh
```

Comment:
Maybe we need to use some wget/curl trick to get the script.

Reply:
Changed.

Reply:
Fixed.